account_invoice_split/models/account_invoice.py

```diff
@@ -0,0 +1,66 @@
+from odoo import fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class AccountInvoiceInherit(models.Model):
+    _inherit = "account.invoice"
+
+    split_id = fields.Many2one(
+        string="Split From",
+        comodel_name="account.invoice",
+        help="INV Split From Ref:",
+    )
+
+    def btn_split_quotation(self):
```

Comment:
I don't understand where this button can be clicked.

Reply:
in account_invoice_view.xml line 17 :-)